backbone/src/main/java/org/researchstack/backbone/model/survey/SubtaskQuestionSurveyItem.java

```diff
@@ -10,27 +10,4 @@
     SubtaskQuestionSurveyItem() {
```

Comment:
Survey only relates to the JSON model.  So this one is a SubtaskStep that contains only questions.  Consent Quiz is a SubtaskQuestionSurveyItem.